packages/graphql/src/classes/subscription/cdc/cdc-api.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Cypher from "@neo4j/cypher-builder";
+import type { Driver } from "neo4j-driver";
+import type { CDCEvent, CDCQueryResponse } from "./cdc-types";
+
+export class CDCApi {
+    private driver: Driver;
+    private token: string = "";
+
+    constructor(driver: Driver) {
+        this.driver = driver;
+    }
+
+    public async queryEvents(): Promise<CDCEvent[]> {
+        if (!this.token) {
+            this.token = await this.fetchCurrentToken();
+        }
+
+        const tokenLiteral = new Cypher.Literal(this.token);
```

Comment:
this constant doesn't seem to bring much value.
It also seems to be an implementation detail of the query procedure, which could be extracted into its own function like the current procedure is such that this function deals with the same level of abstraction. What do you think?

Reply:
So, the idea of the `CDCProcedures` wrapper is to act as the methods that should be in cypher builder eventually. 

This line will remain outside of the cypher builder (at least until the API is revamped there) so that's why it is not within the `CDCProcedures.query`

The reason for having this as a separate line is simply to split the long line making the procedure

Reply:
My bad for being ambiguous, I was referring to another function like `this.fetchCurrentToken()` is for `current`. I get why you wouldn't modify the `query` for this :D